tx-parser/src/lib.rs

```diff
@@ -13,12 +13,17 @@
 extern crate edn;
 extern crate combine;
 extern crate mentat_tx;
+extern crate mentat_parser_utils;
+
+#[macro_use]
+extern crate mentat_query_parser;
 
 use combine::{any, eof, many, optional, parser, satisfy_map, token, Parser, ParseResult, Stream};
 use combine::combinator::{Expected, FnParser};
 use edn::symbols::NamespacedKeyword;
 use edn::types::Value;
-use mentat_tx::entities::*;
+use mentat_tx::entities::{Entid, EntidOrLookupRef, Entity, LookupRef, ValueOrLookupRef};
```

Comment:
There's a clash of styles it would seem, as @ncalexan opted for `*` in the first place. Have we settled on a single style for our codebase? My question was directed to the fact that I didn't see how it this change was related to the issue being fixed (it could've been for what I know, I still don't have a solid grasp on rust).